src/components/proposal/ProposalTable.tsx

```diff
@@ -82,6 +82,14 @@ const ProposalTable = ({
     return <Redirect push to={`/ProposalEdit/${editProposalPk}`} />;
   }
 
+  const showReferenceText = (
+    proposalData: PartialProposalsDataType[]
+  ): boolean => {
+    return proposalData.some((proposal) => {
+      return proposal.call?.referenceNumberFormat && !proposal.submitted;
```

Comment:
I wonder about `&& !proposal.submitted;` part. Isn't STFC proposal editable after submission so the message should be shown until proposal is accepted and reference is updated ?